templates/vue/src/components/tyde/TydeCockpit.vue

```diff
@@ -65,6 +53,12 @@ export default {
       }
       return this.logsWithAnswers
     },
+    cockpitImage() {
+      return {
+        backgroundImage: `url(${this.settings.spaceshipBackgroundUrl})`
+      }
```

Comment:
I'm not able to change this setting anywhere in the UI. Not sure if that's intended. Can you clarify @wynnset ?

Reply:
You can change the setting by going to the Tapestry settings (the gear icon where you also can change the background for the Tapestry or turn auto-layout on/off) and then going to the TYDE tab.